lib/util/fs/index.spec.ts

```diff
@@ -18,7 +20,9 @@ import {
 } from '.';
 
 jest.mock('../../util/exec/env');
+jest.mock('find-up');
 const env = mocked(_env);
+const { findUp } = mocked({ findUp: _findUp });
```

Comment:
```suggestion
const findUp = _findUp as any;
```
Let's fix the types later.